pkg/networkservice/connectioncontext/dnscontext/client.go

```diff
@@ -33,13 +33,16 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/tools/log"
 )
 
+const (
+	modifiedResolvConfKey = "# modified"
```

Comment:
Can we simply try to parse resolv.conf from the comments? 

If the comments can be parsed as resolv.conf then we are using it. Otherwise we may think that the resolv.conf was not stored and we may do store operation.